pandas_gbq/gbq.py

```diff
@@ -15,6 +15,8 @@
 from pandas.core.common import PandasError
 from pandas.compat import lzip, bytes_to_str
 
+logger = logging.getLogger(__name__)
```

Comment:
yep - ideally we test with it (to make sure it works)

and have some examples in docs in how to do it